.gitignore

```diff
@@ -31,6 +31,7 @@ search_index.json
 **/.tern-port
 /.tern-project
 /queries
+/example/app/efs-cf.yml
```

Comment:
Is this for your branch or something that gets generated?

Reply:
That's a file that I have locally.  It's a CF template that will create an EFS volume that I can use for testing the EFS mount functionality in our ECS cluster.